test/modules/shadowing/var-shadows-private-use.chpl

```diff
@@ -0,0 +1,12 @@
+module M {
+  var x: int;
+}
+
+module N {
+  private use M;
+  var x: int;
+
+  proc main() {
+    writeln(x);
+  }
+}
```

Comment:
Nothing about this test is making it clear which `x` is being written, right?  Should the `x`s be given different types or values to discriminate between them better?

Reply:
I've just done so.